README.md

```diff
@@ -46,16 +47,41 @@ const Metrics = require('testpilot-metrics');
 const { sendEvent } = new Metrics({
   id: '@my-addon',
   version: '1.0.2a',
+
+  // These keys are also required if you're using GA:
   uid: 'some-non-PII-user-ID',
   tid: 'UA-XXXXXXXX-YY'
 });
 
+// Sometime later, when a button gets clicked:
 sendEvent({
-  object: 'webext-button',
+  object: 'toolbar-button',
   method: 'click'
 });
 ```
 
+### Generating a `uid` to send to GA
+
+There are many ways to generate a unique, non-PII user ID. Here are some examples:
+
+WebExtension:
+
+```js
+// returns a cryptographically strong random integer
+// example: 3696675532
+const uid = window.crypto.getRandomValues(new Uint32Array(1))[0];
+```
+
+SDK:
+
+```js
+const { uuid } = require('sdk/util/uuid');
+
+// returns a random uuid v4 string
+// example: "b0aef941-35c0-6748-912f-d779c6e4d05e"
+const uid = uuid().number.replace('{','').replace('}','');
```

Comment:
Thanks for the comments :+1: I'm going to leave it as-is, if only because a module named 'util/uuid' sure shouldn't return a uuid wrapped in curly braces. I'd rather people think, "wtf? why remove curly braces...oh" than "wtf? why trim the first and last char off a uuid v4?"